theia-electron/package.json

```diff
@@ -66,14 +66,18 @@
     },
     "devDependencies": {
         "@theia/cli": "next",
-        "electron-builder": "^20.40.2",
+        "electron-builder": "^22.4.1",
         "bufferutil": "4",
         "utf-8-validate": "5"
     },
+    "// Notes:": [
+        "The resolution for `fs-extra` was required due to this: https://spectrum.chat/theia/general/our-theia-electron-builder-app-no-longer-starts~f5cf09a0-6d88-448b-8818-24ad0ec2ee7c"
+    ],
     "resolutions": {
         "vscode-languageserver-protocol": "3.15.0-next.9",
```

Comment:
I have an application for JAVA and the section "resolutions" is not needed anymore with the latest Theia.  In here we still need the fs-extra only

Reply:
I think we added the resolutions to all apps in this repo - maybe better to remove them all in one shot as well vs piecemeal?